event_model/__init__.py

```diff
@@ -180,6 +180,103 @@ def bulk_datum(self, doc):
         self.datum_page(bulk_datum_to_datum_page(doc))
 
 
+class SingleRunDocumentRouter(DocumentRouter):
+    """
+    A DocumentRouter intended to process events from exactly
```

Comment:
Sphinx gives special significance to the first line of the docstring, for display purposes. Can this be one line?